frontend/src/api/usePutData.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Custom hook updating data.
+ * @param {string} url - adress of request
+ * @param {state} requestBody - body od request made by user
```

Comment:
in curly brackets, we should specify a type so it will be Object (you can "Store this object in a state of a component."